argilla/src/argilla/datasets/_export/_disk.py

```diff
@@ -98,20 +99,28 @@ def from_disk(
             workspace_id = client.workspaces.default.id
         dataset_model.workspace_id = workspace_id
 
-        # Get a relevant and unique name for the incoming dataset.
         if name:
             logging.warning(f"Changing dataset name from {dataset_model.name} to {name}")
```

Comment:
why is this a warning and not info? shouldn't we also only warn if the names and workspaces aren't equal?